src/i18n/locale/sv.json

```diff
@@ -1,4 +1,8 @@
 {
+  "components.Settings.Notifications.validationBotAPIRequired": "Du måste ange en Bot API key.",
+  "components.Settings.Notifications.validationChatIdRequired": "Du måste ange en Chat id.",
+  "components.Settings.Notifications.telegramsettingsfailed": "Notifieringsinställningar för Telegram misslyckades att sparas.",
+  "components.Settings.Notifications.telegramsettingssaved": "Notiferingsinställningar för Telegram sparade!",
```

Comment:
Don't include translations for other languages outside of english. Weblate tends to struggle rebasing when we do this. Just update Weblate right after this is merged.